Neos.Flow/Tests/Unit/Reflection/MethodReflectionTest.php

```diff
@@ -44,4 +45,28 @@ public function getParametersReturnsFlowsParameterReflection($dummyArg1 = null,
             self::assertEquals(__CLASS__, $parameter->getDeclaringClass()->getName());
         }
     }
+
+    public function classAndMethodWithAnnotations()
+    {
+        return [
+            [ClassWithAnnotatedMethod::class, 'methodWithTag', ['skipcsrfprotection' => []]],
+            [ClassWithAnnotatedMethod::class, 'methodWithTagAndComment', ['skipcsrfprotection' => ['Some comment']]],
+            [ClassWithAnnotatedMethod::class, 'methodWithAnnotation', ['flow\skipcsrfprotection' => []]],
+            [ClassWithAnnotatedMethod::class, 'methodWithAnnotationAndComment', ['flow\skipcsrfprotection' => ['Some comment']]],
+            [ClassWithAnnotatedMethod::class, 'methodWithAnnotationArgument', ['flow\validate' => ['"foo"']]],
+            [ClassWithAnnotatedMethod::class, 'methodWithAnnotationArgumentAndComment', ['flow\validate' => ['"foo"']]],
```

Comment:
```suggestion
            [ClassWithAnnotatedMethod::class, 'methodWithAnnotationArgument', ['flow\validate' => ['foo']]],
            [ClassWithAnnotatedMethod::class, 'methodWithAnnotationArgumentAndComment', ['flow\validate' => ['foo']]],
```
since we strip quotes in https://github.com/neos/flow-development-collection/pull/2615/files#diff-941ea85c3cf18e0a178f7351c09b38c976c63ba63f7f0c3130737fb11d0b050aR125